bazel/linux/defs.bzl

```diff
@@ -94,8 +137,12 @@ To create a .tar.gz suitable for this rule, you can use the kbuild tool, availab
             doc = "The name of the downloaded kernel. Format is 'distribution-kernel-version-arch', like debian-5.9.0-rc6-rt-amd64.",
             mandatory = True,
         ),
-        "url": attr.string(
-            doc = "The url to download the package from.",
+        "path": attr.string(
+            doc = "The astore path to download the package from.",
+            mandatory = True,
+        ),
+        "uid": attr.string(
+            doc = "The astore UID for this package.",
```

Comment:
Comment for L#151 further down (Github is lame, doesn't let you comment on unmodified lines).... 

I'd like to drop the `auth` param too, now that it isn't wired up to anything. Will do that in the next rev.